code-studio/build-js.js

```diff
@@ -24,6 +24,7 @@ commander
 // TODO - think about how watchify will work for multiple commands
 build_commands.execute([
   build_commands.ensureDirectoryExists(BUILD_PATH),
+  build_commands.ensureDirectoryExists(BUILD_PATH + 'levels/'),
```

Comment:
I really ought to work this into the factor-bundle command.

TODO: Refactor this file to reduce duplication in the browserify step definitions.